end-to-end-applications/java/food-ordering/webui/src/services/sendToRestate.ts

```diff
@@ -36,8 +36,10 @@ export async function createOrder(orderId: string, order: any) {
 
 export async function getStatus(orderId: string) {
   return await (
-    await axios.post(`${RESTATE_HOST}/OrderStatusService/${orderId}/get`, {
+    await axios.post(`${RESTATE_HOST}/OrderStatusService/${orderId}/get`, null, {
       headers: challengeHeaders(),
+      // not sure why this removes the content-type header
+      transformRequest: (data, headers) => {},
```

Comment:
W/o this, Axios always adds the `content-type` header 🤷

Reply:
admittedly, a POST request without content-type header is a bit weird, but that's an interesting discovery. Perhaps for getStatus use the `GET` verb...

Reply:
We should probably update this code to use the ingress client, as shown here https://github.com/restatedev/sdk-typescript/blob/main/packages/restate-sdk-clients/src/ingress.ts

Reply:
Will leave this as a follow-up to replace the existing code with the typed clients.

Reply:
Replace post with get.